BACKLOG.md

```
@@ -1849,6 +1849,8 @@ jobs:
 
 ## Later (Someday/Maybe, 6+ months)
 
+- **[Reliability] Health deep flag strictness** — Parse `deep` query as `1` only and add edge-case tests (`?deep=0/false/true`); source: PR #7 comment https://github.com/misty-step/gitpulse/pull/7#issuecomment-3568633304
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)